angularjs-portal-home/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -167,6 +174,7 @@ define(['angular', 'jquery'], function(angular, $) {
               //init variables
               $scope.portlets = [];
               marketplaceService.getPortlets().then(function(data) {
+				  console.log(data.portlets);
```

Comment:
That would be a lotta console output, may want to only do that if the `APP_FLAGS.debug` is `true`.